packages/blade/package.json

```diff
@@ -246,16 +254,20 @@
   "bundlemon": {
     "files": [
       {
-        "friendlyName": "Web Components",
-        "path": "./build/components/index.web.js"
+        "friendlyName": "Development Web Components",
```

Comment:
shall we keep a track of dev bundles? this might bloat up bundlemon table?

Reply:
I think we can keep the prod bundle only, initially I added this to check the diff in dev and prod sizes.